python/lsst/dax/imgserv/MetadataFitsDb.py

```diff
@@ -405,15 +386,13 @@ def directoryCrawl(rootDir, metaDb):
 def test(rootDir="~/test_metadata"):
     '''This test only works on specific servers and uses a large dataset.
     '''
-    credFile = "~/.lsst/dbAuth-dbServ.txt"
-    creds = readCredentialFile(credFile, log)
-    dbName = "{}_fitsTest".format(creds['user'])
+    credFile = "~/.lsst/dbAuth-dbServ.ini"
 
     # Destroy existing tables and re-create them
-    dbDestroyCreate(credFile, dbName, "DELETE")
+    dbName = dbDestroyCreate(credFile, "DELETE")
```

Comment:
pylint tells me that `dbDestroyCreate` always returns `None` so assigning it to `dbName` does not make sense, and `dbName` itself is not used anywhere, I think assignment is not needed.